_includes/sign_in.html

```diff
@@ -2,6 +2,6 @@
   class="{{include.mobile}} usa-button sign-in-logo"
   href="{{ site.idp_base_url }}/{% if page.lang!= 'en' %}{{page.lang}}{% endif %}"
   tabindex="0"
-  >{{ site.data.[page.lang].settings.nav.sign_in }}
-  <span>Login.gov</span>
```

Comment:
if anybody else is wondering like me why the heck we need a plain `<span>` here, it's because of these styles
https://github.com/GSA-TTS/identity-site/blob/ce4680eddc868002f2474f63598a8f1871fc0ff2/_sass/components/_nav.scss#L96-L105